eventmon/monitord/impl.go

```diff
@@ -59,7 +67,7 @@ func newMonitor(keymasterServerHostname string, keymasterServerPortNum uint,
 		X509CertChannel:             x509CertChannel,
 		keymasterStatus:             make(map[string]error),
 	}
-	go monitor.monitorForever(logger)
+	go monitor.monitorForever(logger, Teelog)
```

Comment:
Why are you adding a second logger parameter? I was expecting that the teelogger would replace the logger in the parameter list.
What is the goal? To send all logs to both loggers? Or is that not the case?